### DISCUSSION TITLE
can you add a export feature on the website？

### DISCUSSION THREAD

**Discussion by MrFutureV** [openai/whisper#1136] [Q&A] [answered]
https://replicate.com/m1guelpf/whisper-subtitles
i submit my audio and i do get the result，but hopefully it could be a way to export srt file

**Comment by jongwook** [comment, marked as answer]
The linked demo is not affiliated with us.

**Reply by MrFutureV** [reply]
oh coz they have a github redirect to here ,i thought it might be the same one, sorry to bother
